### PR TITLE
x-list -> list

### PR DESCRIPTION
**Pull request by rwwagner90**
Also, converted some things to be angle bracket component syntax. This was just a proof of concept, to ensure things work before we do a larger refactor of the inspector.

### CODE REVIEW

File: app/templates/components/list.hbs [hunk 1]
```diff
@@ -50,7 +49,7 @@ as |content|
       maxWidth=column.maxWidth
       on-drag=(action "didResize" column.id)
       position=(one-way column.width)
-      setIsDragging=setIsDragging
+      setIsDragging=@setIsDragging
```
Comment (pbishop16) [inline app/templates/components/list.hbs]: It seems like some of the test failures point to `@setIsDragging`. Should it remain as `setIsDragging`?

Comment (rwwagner90) [reply app/templates/components/list.hbs]: I think it's that this stuff is incompatible with older Ember versions, not sure how to best handle this

Comment (rwjblue) [reply app/templates/components/list.hbs]: We need to add ember-named-arguments-polyfill I believe...

Comment (rwjblue) [reply app/templates/components/list.hbs]: gah, looks like https://github.com/rwjblue/ember-named-arguments-polyfill only goes down to Ember 2.10 😩

Comment (rwwagner90) [reply app/templates/components/list.hbs]: @rwjblue so adding that polyfill gets things passing on 2.10+. It seems like versions before that still have the error. Thoughts?

Comment (rwwagner90) [reply app/templates/components/list.hbs]: Heh, I needed to refresh my page, you noticed the same things first 😛

**Comment by rwwagner90** [comment]
@rwjblue what should we do for supporting angle bracket components here? Should we drop the tests for anything pre 2.10?

**Pull request merged**